StortSpelProjekt/Game/src/Gamefiles/Components/UpgradeComponents/Upgrades/UpgradeRangeVelocity.cpp

```diff
@@ -41,5 +41,5 @@ void UpgradeRangeVelocity::ApplyBoughtUpgrade()
 
 std::string UpgradeRangeVelocity::GetDescription(unsigned int level)
 {
-	return "Range Velocity: Increases velocity of the range attacks with 10 per level. Max velocity is 100, at upgrade level 5";
+	return "Range Velocity: Increases velocity of the range attacks with 30 per level. Max velocity is 300, at upgrade level 5";
```

Comment:
Hold up! The overall velocity increase makes projectiles to sometimes go through walls. I was told because of the collision detection update rate(0.01?) max velocity should be 100 before collision detection starts to get iffy. Something has happened to the range velocity in the recent merges though so I have no idea what the limits should be now :> at 150 it SOMETIMES goes into walls, at 300 it almost always does. Not sure how to resolve this issue though?

Reply:
I might add that I noticed that collision seems to work fine when the bouncy upgrade is bought, as in the projectiles no longer goes through walls even on 300

Reply:
Does it go INTO walls or does it go THROUGH walls and out on the other side? IT should disappear when colliding with a wall, but due to a mismatch of bullet and our boundingboxes it doesn't always. Can look at a quick fix right away if necessary

Reply:
Hmm ok then the "problem" is that the projectiles bounce on the walls without the upgrade? Idk lets take a couple of minutes and talk about this after lunch :>